src/main/c/Objects/pyjconstructor.c

```diff
@@ -113,27 +113,51 @@ int PyJConstructor_Check(PyObject* object)
 static PyObject* pyjconstructor_call(PyJMethodObject *self, PyObject *args,
                                      PyObject *keywords)
 {
-    PyObject      *firstArg    = NULL;
-    PyJObject     *clazz       = NULL;
-    JNIEnv        *env         = NULL;
-    int            pos         = 0;
-    jvalue        *jargs       = NULL;
-    int           foundArray   = 0; /* if params includes pyjarray instance */
-    PyThreadState *_save       = NULL;
-    jobject   obj  = NULL;
-    PyObject *pobj = NULL;
+    JNIEnv        *env              = NULL;
+    Py_ssize_t     lenPyArgsGiven   = 0;
+    int            lenJArgsExpected = 0;
+    /* The number of normal arguments before any varargs */
+    int            lenJArgsNormal   = 0;
+    PyObject      *firstArg         = NULL;
+    PyJObject     *clazz            = NULL;
+    int            pos              = 0;
+    jvalue        *jargs            = NULL;
+    /* if params includes pyjarray instance */
+    int            foundArray       = 0;
+    jobject        obj              = NULL;
+    PyObject     *pobj              = NULL;
```

Comment:
Totally nitpicky, you can ignore this comment if desired, this last one *pobj is not aligned with the others (needs one more space).

Reply:
Fixed